chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/urlStatus/QuestionableUrlReportGenerator.java

```diff
@@ -310,9 +299,9 @@ private File getOutputFile(List<FailedUrlResult> urlResultsToWrite, String repor
         return Arrays.asList(CSV_HEADER);
     }
 
-    private List<String> generateRowValue(FailedUrlResult urlResult, boolean firstUrlInGroup) {
+    private List<String> generateRowValue(FailedUrlResult urlResult) {
         List<String> result = new ArrayList<String>();
-        if (firstUrlInGroup) {
+        
             result.add(urlResult.getUrl());
```

Comment:
If these lines, down to the four removed "result.add("")" lines, were inside an if, but haven't changed, then their indentation is probably incorrect too